### PR TITLE
(RE-11573) Add fpm gem and hard-code rubylibdir + prep for 1.8.9 release

### PR DESCRIPTION
**Pull request by mwaggett**
see individual commit messages for details

### CODE REVIEW

**Comment by mwaggett** [comment]
Built puppetserver with existing pipeline: https://jenkins-master-prod-1.delivery.puppetlabs.net/job/platform_various-packaging-jobs_packaging-os-clj_lein-ezbake-generic/1096/

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+## 1.8.9 - 2018-11-02
+This is a maintenance release.
+
+Maintenance:
+  * Add fpm to Gemfile that gets included in package builds.
+  * Hard-code `rubylibdir` variable in install script
```
Comment (sarameisburger) [inline CHANGELOG.md]: suggestion: maybe a note as to why we had to hardcode it in? (also missing a period)

Comment (mwaggett) [reply CHANGELOG.md]: That info is in the commit message. I'm never totally sure what level of detail to include here vs. there..

Comment (mwaggett) [reply CHANGELOG.md]: I can try to include a short summary here.

File: resources/puppetlabs/lein-ezbake/template/global/install.sh.erb [hunk 2]
```diff
@@ -232,7 +232,7 @@ function task_termini {
     #
     # This also allows us to prefer PE ruby over system ruby, so this will work
     # with PE as well
-    local rubylibdir=${rubylibdir:=$(/opt/puppetlabs/puppet/bin/ruby -rrbconfig -e "puts RbConfig::CONFIG['vendordir']")}
+    local rubylibdir=${rubylibdir:=/opt/puppetlabs/puppet/lib/ruby/vendor_ruby}
```
Comment (e-gris) [inline resources/puppetlabs/lein-ezbake/template/global/install.sh.erb]: Minor: this has the expected behavior but is redundant

```${rubylibdir:=/opt/puppetlabs/puppet/lib/ruby/vendor_ruby}```

will set the global ```rubylibdir``` variable by itself, so

```local rubylibdir=${rubylibdir:=/opt/puppetlabs/puppet/lib/ruby/vendor_ruby}```

will set the *local* rubylibdir to the value of the global rubylibdir

I suggest:
```local rubylibdir=${rubylibdir:-/opt/puppetlabs/puppet/lib/ruby/vendor_ruby}```

However, note that this only sets the local rubylibdir, so once the function is done, the global rubylibdir goes back into scope.

Comment (e-gris) [inline resources/puppetlabs/lein-ezbake/template/global/install.sh.erb]: I had a comment here but I deleted if after considering the implication. This is good as written.

Comment (mwaggett) [reply resources/puppetlabs/lein-ezbake/template/global/install.sh.erb]: oh haha, I was like, why is github not showing me the comment I received an email about??

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,11 @@
+## 1.8.9 - 2018-11-02
+This is a maintenance release.
+
+Maintenance:
+  * Add fpm to Gemfile that gets included in package builds.
+  * Hard-code `rubylibdir` variable in install script to avoid calling puppet's
+    ruby, which doesn't include bundled dependencies.
```
Comment (mwaggett) [inline CHANGELOG.md]: @sarameisburger updated!

Comment (sarameisburger) [reply CHANGELOG.md]: yay thanks!

**Pull request merged**